### PR TITLE
[PHP] Added kuiper framework

### PR DESCRIPTION
**Pull request by wenbinye**
Kuiper is a simple php framework, document by https://kuiper-framework.github.io/en/

### CODE REVIEW

File: php/kuiper/composer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "require": {
+    "php": "^8.1",
+    "ext-json": "*",
+    "kuiper/web": "^0.8"
```
Comment (waghanza) [inline php/kuiper/composer.json:5]: to align constraint on php based frameworks

```suggestion
    "kuiper/web": "~0.8"
```

File: php/kuiper/composer.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "require-dev": {
+    "kuiper/component-installer": "^0.1"
+  },
```
Comment (waghanza) [inline php/kuiper/composer.json:9]: do we need this ?

File: php/kuiper/config.yaml
Comment (waghanza) [inline php/kuiper/config.yaml:1]: don't kuiper require pcntl ?

File: php/kuiper/composer.json (hunk 1 above)
Comment (wenbinye) [reply php/kuiper/composer.json:5]: kuiper framework is not released currently. ^0.8 equal to >=0.8 and < 0.9, ~0.8 equal to >= 0.8 and < 1.0，previous one is prefered

File: php/kuiper/composer.json (hunk 2 above)
Comment (wenbinye) [reply php/kuiper/composer.json:9]: not require at runtime, only for development

Comment (waghanza) [reply php/kuiper/composer.json:9]: ok, maybe use `@stable` then :-)

**Review by waghanza — COMMENTED** [review]
I have a notice 
```
[2023-03-21 09:31:29 *17.2]	NOTICE	ProcessFactory::end() (ERRNO 1005): session#42 does not exists
```
when running instead the container

File: php/kuiper/composer.json (hunk 1 above)
Comment (waghanza) [reply php/kuiper/composer.json:5]: you are right, `composer` act differently for versions behind 1.0. maybe using `~0.8.0` will be clearer then

File: php/kuiper/config.yaml
Comment (wenbinye) [reply php/kuiper/config.yaml:1]: I think it is not necessary. Swoole can handle signal well.

**Review by waghanza — APPROVED** [review]

**Pull request merged**